.gitattributes

```diff
@@ -0,0 +1,52 @@
+*.doc  diff=astextplain
+*.DOC	diff=astextplain
+*.docx	diff=astextplain
```

Comment:
Why whitespace so inconsistent?

Reply:
This is just a direct copy of what we have in [aspnet/Mvc](https://github.com/aspnet/Mvc/blob/release/2.2/.gitattributes). Which [other repos](https://github.com/aspnet/KestrelHttpServer/blob/release/2.2/.gitattributes) also directly ripped off.